UploadAndAnnotateImages/README.md

```diff
@@ -16,7 +16,7 @@ We recommend taking the [Quick Start: Lightning Web Components](https://trailhea
    1. From [LWC-Mobile-Samples](https://github.com/forcedotcom/LWC-Mobile-Samples), click **Code**.
    2. Clone the code or download the zip file.
 2. Configure the LWC:
-   1. Open the project in Visual Studio Code.
+   1. Open the UploadAndAnnotateImages folder in Visual Studio Code.
```

Comment:
Nit: Given the convention in other immediate parts of the README, where we annotate file/folder paths as `code`, can we do the same for `UploadAndAnnotateImages` here?

Reply:
Good point, thanks!